packages/widgetbook/lib/src/fields/field.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/src/fields/field_codec.dart';
+import 'field_type.dart';
+
+abstract class Field<T> {
+  const Field({
+    required this.group,
+    required this.name,
+    required this.type,
+    required this.codec,
+    required this.onChanged,
+  });
+
+  final String group;
+
+  /// Name of this in the query parameter group.
```

Comment:
Did you mean "Name of this Field in ..."?

Reply:
I see that you used "this" consistantly. I find it confusing. You can either do `[this]` instead of just `this` in comments or go with the actual name of the class.